transport/x/grpc/inbound.go

```diff
@@ -85,22 +91,17 @@ func (i *Inbound) start() error {
 	if i.router == nil {
 		return errRouterNotSet
 	}
-	serviceDescs, err := i.getServiceDescs()
-	if err != nil {
-		return err
-	}
+
+	i.handler = newHandler(
```

Comment:
I might be missing something, is there any reason this handler needs to be stored as a variable on the Transport?